### PR TITLE
[azure] add workflow for compact 3-node cluster on azure public cloud

### PR DESCRIPTION
**Pull request by jinyunma**
[CORS-2419](https://issues.redhat.com/browse/CORS-2419) Compact 3-node Clusters on Public Cloud (Azure)

### CODE REVIEW

**Comment by jinyunma** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.13-installer-rehearse-azure

**Comment by jinyunma** [comment]
@jianlinliu @gpei PTAL, thanks.

File: ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/compact-3node/cucushift-installer-rehearse-azure-ipi-compact-3node-workflow.metadata.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+	"path": "cucushift/installer/rehearse/azure/ipi/compact-3node/cucushift-installer-rehearse-azure-ipi-compact-3node-workflow.yaml",
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/compact-3node/cucushift-installer-rehearse-azure-ipi-compact-3node-workflow.metadata.json:2]: suggest to rename `compact-3node` to `compact`

**Pull request merged**